hr_employee_relative/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
```

Comment:
It's common nowadays to remove licensing and copyright from `__init__.py` files that amount just a few imports.

Reply:
Copyright is usually removed, but not license, which should be in all files as stated by the AGPL license.